crates/node/src/lib.rs

```diff
@@ -32,35 +38,37 @@ pub struct ScanResult {
 #[napi]
 impl ScanResult {
   #[napi]
-  pub fn scan_files(&self, input: Vec<ChangedContent>) -> Vec<String> {
-    tailwindcss_oxide::scan_files_with_globs(
-      input.into_iter().map(Into::into).collect(),
-      self.globs.clone().into_iter().map(Into::into).collect(),
-    )
+  pub fn scan_files(&self, _input: Vec<ChangedContent>) -> Vec<String> {
+    let result = tailwindcss_oxide::scan_dir(tailwindcss_oxide::ScanOptions {
+      base: self.base.clone(),
+      sources: self.sources.clone().into_iter().map(Into::into).collect(),
+    });
+
+    result.candidates
   }
```

Comment:
Public API uses `scanDirResult.scanFiles()` for incremental rebuilds. Right now we will do a full scan behind the scenes.

Good idea, @philipp-spiess!